### PR TITLE
[FSDE-16823] Improve printing methods for child and grand child rows in data table

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**
https://ctl.atlassian.net/browse/FSDE-16823

### CODE REVIEW

File: src/chi-vue/src/components/data-table/DataTable.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default class DataTable extends Vue {
   treeSelection = Object.prototype.hasOwnProperty.call(this.$props.config, 'treeSelection')
     ? this.$props.config.treeSelection
     : defaultConfig.treeSelection;
-  printMode = this.$props.config.printMode || defaultConfig.printMode;
+  printMode = this.$props.config?.print?.mode || defaultConfig.print?.mode;
```
Comment (janibolkvadze) [inline src/chi-vue/src/components/data-table/DataTable.tsx:65]:
```suggestion
  printMode = this.$props.config?.print?.mode || defaultConfig.print.mode;
```
We control the `defaulctConfig` so it must be set always 😉

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-354/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://c673d4e3b7f9272cc83fb4b73f4852fb080060ab-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-354-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-354/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://8776c75fed0266a0abbc170e5bc49ee574d4940e-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-354-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Pull request merged**